actor/builtin/miner/miner_test.go

```diff
@@ -1133,6 +1146,30 @@ func TestMinerGetPoStState(t *testing.T) {
 	})
 }
 
+func TestMinerGetSlashable(t *testing.T) {
+	tf.UnitTest(t)
+
+	firstCommitBlockHeight := uint64(3)
+
+	t.Run("is reported as slashable if it has a proving set", func(t *testing.T) {
+		mal := setupMinerActorLiason(t)
+		mal.requireCommit(firstCommitBlockHeight, uint64(1))
+		mal.requireCommit(firstCommitBlockHeight+1, uint64(2))
+		mal.requireCommit(firstCommitBlockHeight+2, uint64(17))
+
+		// submit PoSt to update proving set with no done sectors
+		done := types.EmptyIntSet()
+		mal.requirePoSt(firstCommitBlockHeight+5, done)
+		// even though they are late, they have never submitted a post so
+		mal.assertSlashability(true, firstCommitBlockHeight+6)
+	})
+
+	t.Run("is reported as not slashable if no proving set", func(t *testing.T) {
+		mal := setupMinerActorLiason(t)
+		mal.assertSlashability(false, firstCommitBlockHeight)
+	})
```

Comment:
It might also be good to test the edge case where a miner has a single sector in its proving set after its first commitSector but before its first submitPoSt

Reply:
I thought that was the last example. What's missing?

Reply:
It's similar to the last example but you would run `mal.assertIsProving(true. firstCommitBlockHeight+1)` after the first `mal.requireCommit`

Reply:
ah. I was thinking about storagemarket_test where I'm doing that already, but I'll put an explicit one here.